docarray/typing/tensor/tensor.py

```diff
@@ -23,9 +24,9 @@ def __get_validators__(cls):
     @classmethod
     def validate(
         cls: Type[T],
-        value: Union[T, np.ndarray, List[Any], Tuple[Any], Any],
-        field: 'ModelField',
-        config: 'BaseConfig',
+        value: Union[Any, Any, Any],
```

Comment:
same here lets keep the old line other we lose info

Reply:
Because if do so, the mypy will raise the error about the incompatibility between abstract class and inherided type.